File: abc/abc231/b.cpp
// https://atcoder.jp/contests/abc231/submissions/27820546

#include <bits/stdc++.h>
using namespace std;

int main(){
    int N, max = 0;
    string S, ans;
    map<string, int> m;
    cin >> N;
    for (int i = 0; i < N; i++) 
    {
        cin >> S;
        m[S]++;
        if (max < m[S]) {
            max = m[S];
            ans = S;
        }
    }
    cout << ans << endl;
    return 0;
}

File: abc/abc231/b.py
# https://atcoder.jp/contests/abc231/submissions/37705646

n = int(input())
dist = {}

for _ in range(n):
  s = input()
  if s in dist:
    dist[s] += 1
  else:
    dist[s] = 1

name = ""
ma = -1e9

for k, v in dist.items():
  if ma < v:
    name = k
    ma = v

print(name)
